app/bundles/CoreBundle/Helper/Chart/SeriesPieChart.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+namespace Mautic\CoreBundle\Helper\Chart;
+
+class SeriesPieChart extends AbstractChart implements ChartInterface
+{
+    /**
+     * Holds the suma of the all dataset values.
+     *
+     * @var float
+     */
+    protected $totalCount = 0;
+
+    /**
+     * @return array{labels: mixed[], datasets: mixed[]}
+     */
+    public function render(bool $withCounts = true): array
+    {
+        $dataset   = [];
+
+        foreach ($this->datasets as $datasetId => $value) {
+            $data        = ['data' => [], 'backgroundColor' => [], 'hoverBackgroundColor' => []];
+
+            foreach ($value as $key => $item) {
+                $color = $this->configureColorHelper($key);
+
+                $data['data'][]                 = $item;
+                $data['backgroundColor'][]      = $color->toRgba(0.8);
+                $data['hoverBackgroundColor'][] = $color->toRgba(0.9);
+            }
+
+            $data['label'] = $this->labels[$datasetId];
+            $dataset[]     = $data;
+        }
+
+        return [
+            'labels'   => $this->labels,
```

Comment:
To be on a safer side, 

```suggestion
            'labels'   => $this->labels ?? [],
```

Reply:
I changed the declaration to 


`protected $labels = [];`